ui/assets/lang/locale-en.json

```diff
@@ -65,5 +65,7 @@
   "helpMenuLabel": "Help",
   "userGuideLabel": "User Guide",
   "userGuideUrl": "https://github.com/wayfair-contribs/plentymarkets-plugin/blob/main/meta/documents/user_guide_en.md",
-  "clickTheHelpButton": "Click Help for more information"
+  "clickTheHelpButton": "Click Help for more information",
+  "empty_fields": "Empty fields are NOT allowed.",
+  "duplicate_warehouses": "A Plentymarkets Warehouse should be assigned to at most one Wayfair Supplier ID"
```

Comment:
Wasn't mapping multiple warehouses to one supplier ID a feature we enabled at Anish's request?

Reply:
Close. a supplier ID can be mapped to multiple warehouses (ex: one supplier has a west coast warehouse and and east coast warehouse). We do NOT support multiple supplier IDs per Warehouse (ex: supplierA and supplierB both ship things out of the same Warehouse), because we would never know which supplier ID to tag inventory with.

Reply:
👍